admin_guide/scheduling/node_affinity.adoc

```diff
@@ -0,0 +1,281 @@
+[[admin-guide-sched-affinity]]
+= Using Node Affinity
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+
+_Node affinity_ is a set of rules used by the scheduler to determine which nodes are selected for a pod. The rules are defined using custom xref:../architecture/core_concepts/pods_and_services.adoc#labels[labels on nodes] and selectors specified in pods.
+
+Node affinity allows you to constrain which nodes your pod is eligible to be scheduled on using labels on the node. For example, you could configure a pod to only run on a node with a specific CPU or in a specific availability zone.
```

Comment:
Is the first sentence saying anything different form the first paragraph? It seems redundant. the example is good though.